clients/standard_client/client.go

```diff
@@ -24,8 +24,21 @@ var (
 	ErrServiceUnavailable = fmt.Errorf("eigenda service is temporarily unavailable")
 )
 
+// EncodingType represents the serialization format for the certificate
+type EncodingType string
```

Comment:
maybe `CertEncoding` or `CertSerialization` instead?

Reply:
Also not a fan of defining fundamental types in the client. Could we define these in some proxy package and import it in the client instead?

Reply:
how would you do that in a way that doesn't introduce proxy's dependency graph to the client?

Reply:
You import a package that doesn't have many transitive dependencies. For example I tried adding a main.go in clients that only imports common/consts:
```
package clients

import (
	"fmt"

	"github.com/Layr-Labs/eigenda-proxy/common/consts"
)

func main() {
	fmt.Println(consts.EthHappyPathFinalizationDepthBlocks)

}
```

I don't fully understand it but I think golang does lazy module loading, and only based on PACKAGES that a build requires. The difference between package and module is important here. See https://go.dev/ref/mod#go-mod-file-go:
> The go.mod file includes an explicit [require directive](https://go.dev/ref/mod#go-mod-file-require) for each module that provides any package transitively imported by a package or test in the main module.

Reply:
gotcha but wouldn't there by be a cyclic issue here? https://github.com/Layr-Labs/eigenda-proxy/blob/main/go.mod#L30-L31

Reply:
server module only imports clients for testing right? I think that's allowed...

Reply:
see https://github.com/Layr-Labs/eigenda-proxy/pull/376 for an example